internal/admin/siginfo/view.go

```diff
@@ -45,12 +45,12 @@ func (v *viewController) Execute(c *gin.Context) {
 	} else {
 		m.AddJumbotron("Signature Info", "Edit Signature Info")
 
-		exportDB := database.New(v.env.Database())
```

Comment:
Just want to double check that this move is cosmetic. If it's not, can we document why exportDB has to come after the ParseInt?

Reply:
it's cosmetic.

but if you're going to error out, no reason to init the DB